src/screens/RestoreSheet.js

```diff
@@ -95,30 +95,34 @@ export default function RestoreSheet() {
     });
   }, [goBack, navigate]);
 
-  const wrapperHeight =
-    deviceHeight + longFormHeight + (android ? getSoftMenuBarHeight() / 2 : 0);
```

Comment:
@BrodyHughes we need your Samsung phone

Reply:
<img width="514" alt="Screenshot 2022-10-21 at 14 20 08" src="https://user-images.githubusercontent.com/16062886/197273338-209a40c1-71d8-47d6-9833-e81bef943e38.png">
It works pretty well